dev/archery/archery/crossbow/reports.py

```diff
@@ -121,6 +121,61 @@ def show(self, outstream, asset_callback=None):
                                    asset))
 
 
+class JsonReport(Report):
```

Comment:
We don't have many tests yet for crossbow, but this looks like one we could actually test.

You can [reuse the test case including the fixture](https://github.com/apache/arrow/blob/master/dev/archery/archery/crossbow/tests/test_reports.py#L24-L35) from `crossbow/test_reports.py`.